Amplify/Categories/Auth/AuthCategoryBehavior.swift

```diff
@@ -80,6 +82,11 @@ public protocol AuthCategoryBehavior {
                          listener: AuthWebUISignInOperation.EventListener?) -> AuthWebUISignInOperation
 
     /// SignIn using an auth provider on a web UI
+    ///
+    /// Calling this method directly calls the authProvider's signIn interface in a webUI. Depending on the plugin
+    /// implementation and the authentication state with the provider, this method might complete without showing
+    /// any UI.
```

Comment:
"Calling this method will invoke the AuthProvider's default web user interface. Depending ..."